pkg/deployer/clusterapiserver.go

```diff
@@ -73,8 +73,13 @@ func getApiServerCerts() (*caCertParams, error) {
 	return certParams, nil
 }
 
-// GetApiServerYaml returns the clusterapi-apiserver manifest used for deployment
+// GetApiServerYaml returns the cluster-apiserver manifest for deployment in the default namespace
 func GetApiServerYaml() (string, error) {
+	return GetApiServerYamlForNamespace(corev1.NamespaceDefault)
+}
+
+// GetApiServerYamlForNamespace returns the clusterapi-apiserver manifest used for deployment in the supplied namespace
+func GetApiServerYamlForNamespace(namespace string) (string, error) {
```

Comment:
There is also [a function](https://github.com/kubernetes-sigs/cluster-api/blob/81c3f14b2f143e4f962c2420bc0ecba1fd39f7ed/pkg/deployer/clusterapiserver.go#L46-L74) in this file, used by this one, that seems to depend on the default namespace. Maybe it should be updated as well.

Reply:
Yes, I missed that.